sentinelhub/data_collections.py

```diff
@@ -167,26 +167,25 @@ class DataCollectionDefinition:
     # The following parameter is used to preserve custom DataCollection name during pickling and unpickling process:
     _name: Optional[str] = field(default=None, compare=False)
 
-    def __post_init__(self):
+    def __post_init__(self):  # type: ignore
```

Comment:
I'm ignoring the fact that it is not annotated. If i annotate it it cases all kinds of issues in the text of the function, which cannot be easily resolved (because we're doing python magic)